modules/fedimint-mint-client/src/oob.rs

```diff
@@ -49,6 +69,20 @@ pub struct MintOOBStatesCreated {
     pub(crate) timeout: SystemTime,
 }
 
+#[derive(Debug, Clone, Eq, PartialEq, Hash, Decodable, Encodable)]
+pub struct MintOOBStatesCreatedMulti {
+    pub(crate) spendable_notes: Vec<(Amount, SpendableNote)>,
+    pub(crate) timeout: SystemTime,
+}
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash, Decodable, Encodable)]
+pub struct MintOOBStatesUserRefundMulti {
+    /// The txid we are hoping succeeds refunding all notes in one go
+    pub(crate) refund_txid: TransactionId,
+    /// The notes we are going to refund individually if it doesn't
+    pub(crate) spendable_notes: Vec<(Amount, SpendableNote)>,
```

Comment:
Do we still need the spendable notes here now that we discovered the mint input SMs belonging to the refund tx can just take care of this?

If we can avoid that one field we don't need `UserRefundMulti` anymore because the semantics should be the same as the old one (just that now the refund tx contains multiple notes). As a second order effect we could also get rid of the now obsolete `Created` state. 

The reconciliation logic in the SM migration might be more than we want to stomach right now though :/ (we'd need to rewrite SM history such that the update stream still works properly). But we'll have to deal with that eventually anyway I fear. Thoughts @dpc?

Reply:
@elsirion 

I think it's just easier to let the old states be as they are, and then after like two releases replace their code with `Err("No longer supported")`. If we start doing complex migrations, then we should have some (probably also complex) tests, etc.